src/osx/cocoa/textctrl.mm

```diff
@@ -1053,6 +1054,11 @@ - (void)controlTextDidEndEditing:(NSNotification *)aNotification
             fgcolor = [storage attribute:NSForegroundColorAttributeName atIndex:position effectiveRange:NULL];
             ultype = [storage attribute:NSUnderlineStyleAttributeName atIndex:position effectiveRange:NULL];
             ulcolor = [storage attribute:NSUnderlineColorAttributeName atIndex:position effectiveRange:NULL];
+            NSDictionary *attrs;
+            NSRange effectiveRange;
```

Comment:
This seems to be unused.

Reply:
@vadz 
effectiveRange variable is used as a parameter passed to attributesAtIndex.